bindata/v4.1.0/kube-controller-manager/leader-election-cluster-policy-controller-rolebinding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  namespace: openshift-kube-controller-manager
+  name: system:openshift:leader-lock-cluster-policy-controller
```

Comment:
doesn't this name conflict with the openshift one?

Reply:
> doesn't this name conflict with the openshift one?

ha, it doesn't.  Still make a different role and binding.

Reply:
yup, will clean that up